xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1030 @@
+    </para>
+    <para>
+     We highly recommend running separate helm chart and container registry
+     mirrors for each source registry.
+    </para>
+   </note>
+   <para>
+    Container image registries are provided by SUSE, Docker, and other sources.
+    The &suse; container registry is used to update the &productname;
+    components.
+   </para>
+   <para>
+    Once configured, you will not have to modify Dockerfile(s) or Kubernetes
+    manifests to use the mirrors. The requests are passed through the container
+    engine which forwards them to the configured mirrors. For example: All
+    images with a prefix <literal>registry.suse.com/</literal> will be
+    automatically pulled from the configured (internal) mirror.
+   </para>
+   <para>
+    For further information on registry mirror configuration, refer to
+    <xref linkend="sec.admin.velum.registry" /> and
+    <xref linkend="sec.admin.velum.mirror" />.
+   </para>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.container-registry.mirror">
+    <title>Mirror Configuration</title>
+    <note>
+     <title>Internal Registry Mirror Is Read Only</title>
+     <para>
+      For security reasons, the internal registry mirror is configured in
+      <literal>read-only</literal> mode. Therefore, pushing container images to
+      these mirrors will not be possible. They can only serve images that were
+      previously pulled and cached by the external mirror and then uploaded to
+      the internal mirror.
+     </para>
+     <para>
+      If you need the ability to store modified container images, you will have
+      to create a new registry that will hold these images. The steps needed
+      to run your own full container image registry are not part of this
+      document.
+     </para>
+      <para>
+      For more information you can refer to:
+      <link xlink:href="https://docs.docker.com/registry/">Docker Registry</link>.
+     </para>
+    </note>
+
+    <para>
+     On the external mirror server perform the following steps.
+    </para>
+
+    <procedure>
+     <title>Set Up The External Mirror</title>
+     <step>
+      <para>
+       Install <filename>helm-mirror</filename> and <filename>skopeo</filename>.
+      </para>
+<screen>&prompt.root;zypper in helm-mirror skopeo
+      </screen>
+     </step>
+     <step>
+      <para>
+       Install the <filename>docker-distribution-registry</filename> package:
+      </para>
+<screen>&prompt.root;<command>zypper in docker-distribution-registry</command>
+   </screen>
+     </step>
+     <step>
+      <para>
+       Create basic authentication credentials for registry. Replace
+       <literal>USERNAME</literal> and <literal>PASSWORD</literal> with
+       proper credentials of your choosing.
+      </para>
+<screen>&prompt.user;<command>mkdir auth</command>
+&prompt.user;<command>docker run --entrypoint htpasswd registry:2.6.2 -Bbn <replaceable>USERNAME PASSWORD</replaceable> > auth/htpasswd</command>
+ </screen>
```

Comment:
I think this is still pending correct? The fact that we assume the user does not have docker running means that the instructions on section `PROCEDURE 2.5: SET UP THE EXTERNAL MIRROR` are not correct because they mention `docker save...`

Reply:
but that is just the `docker service`. Once teh docker service is running he can run all the docker commands. maybe just moving step 8 to the top, to ensure the client will have docker running.

Reply:
Yes it does provide `htpasswd`